LOG.tbasicu

```diff
@@ -190,150 +198,150 @@ Type tLogThis
       end if
       
-      if Bit_Get(logWhere,3) then ' %LT_ToConsoleTitle is SET
+      if me.private_isFlagSet(logWhere, %LT_ToConsoleTitle) then
         Console_SetTitle(sDT & sTag & LogWhatTitle & LogWhat)
       end if
       
-      if Bit_Get(logWhere,4) then ' %LT_ToMsgBox is SET
+      if me.private_isFlagSet(logWhere, %LT_ToMsgBox) then
         MsgBoxW(0,logWhat, me.msgboxStyle + Switch(severityTag <= %TAG_ERR, %MB_ICONERROR, severityTag = %TAG_WARN, %MB_ICONWARNING, %TRUE, %MB_ICONINFORMATION), sDT & sTag & logWhatTitle)
       end if
       
       if len(logWhatTitle) > 0 then logWhatTitle &= me.titleSeparator
       
-      if Bit_Get(logWhere,2) then ' %LT_ToConsole is SET
+      if me.private_isFlagSet(logWhere, %LT_ToConsole) then
         Print sDT IN IIf(severityTag <= me.minSeverityHighlight, me.ColorTime OR me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, me.ColorTime OR me.ColorBackground)
         Print sTag in IIf(severityTag <= me.minSeverityHighlight, me.ColorTag OR me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, me.ColorTag OR me.ColorBackground) 
         
-'[t] Workaround for PrintL error ( see https://www.thinbasic.com/community/project.php?issueid=617 ) ...
+      #region "Workaround for PrintL error"
+      '[t] Workaround for PrintL error ( see https://www.thinbasic.com/community/project.php?issueid=617 ) ...
         
         'PrintL((LogWhatTitle & LogWhat) IN (Switch(Bit_Get(logWhere,0), me.ColorDebugText Or me.ColorBackground, severityTag <= me.minSeverityHighlight and Bit_Get(logWhere,0), me.ColorDebugText Or me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, severityTag <= me.minSeverityHighlight, me.ColorText Or me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, %TRUE, me.ColorText Or me.ColorBackground)))
         'PrintL(LogWhatTitle & LogWhat IN %CONSOLE_FOREGROUND_BLUE)'(Switch(Bit_Get(logWhere,0), me.ColorDebugText Or me.ColorBackground, severityTag <= me.minSeverityHighlight and Bit_Get(logWhere,0), me.ColorDebugText Or me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, severityTag <= me.minSeverityHighlight, me.ColorText Or me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, %TRUE, me.ColorText Or me.ColorBackground)))
        
         myColor = Switch(Bit_Get(logWhere,0), me.ColorDebugText Or me.ColorBackground, severityTag <= me.minSeverityHighlight and Bit_Get(logWhere,0), me.ColorDebugText Or me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, severityTag <= me.minSeverityHighlight, me.ColorText Or me.ColorBackground Or %CONSOLE_FOREGROUND_INTENSITY, %TRUE, me.ColorText Or me.ColorBackground)
         PrintL(LogWhatTitle & LogWhat IN myColor)
         
-' ... workaround
+      #endregion
     
       end if
       
-      if Bit_Get(logWhere,5) then ' %LT_ToLogFile is SET
-        if me.RotateLog = %FALSE then allOK = allOK and %FALSE
+      if me.private_isFlagSet(logWhere, %LT_ToLogFile) then
+        if me.RotateLog = %FALSE then allOK = %FALSE
```

Comment:
```suggestion
        if me.RotateLog = %FALSE then allOK = allOK and %FALSE
```
here we do not test if it's allOK, therefore we shall merge the boolean with previous state. (yes, for now there isn't but just in case of further development)